types/react/index.d.ts

```diff
@@ -1184,26 +1219,26 @@ declare namespace React {
         /**
          * The font property is shorthand that allows you to do one of two things: you can either set up six of the most mature font properties in one line, or you can set one of a choice of keywords to adopt a system font setting.
          */
-        font?: CSSWideKeyword | any;
+        font?: CSSWideKeyword | string;
 
         /**
          * The font-family property allows one or more font family names and/or generic family names to be specified for usage on the selected element(s)' text. The browser then goes through the list; for each character in the selection it applies the first font family that has an available glyph for that character.
          */
-        fontFamily?: CSSWideKeyword | any;
+        fontFamily?: CSSWideKeyword | string;
 
         /**
          * The font-kerning property allows contextual adjustment of inter-glyph spacing, i.e. the spaces between the characters in text. This property controls <bold>metric kerning</bold> - that utilizes adjustment data contained in the font. Optical Kerning is not supported as yet.
          */
-        fontKerning?: CSSWideKeyword | any;
+        fontKerning?: CSSWideKeyword | "auto" | "normal" | "none";
 
         /**
          * Specifies the size of the font. Used to compute em and ex units.
          * See CSS 3 font-size property https://www.w3.org/TR/css-fonts-3/#propdef-font-size
          */
         fontSize?: CSSWideKeyword |
-                   "xx-small" | "x-small" | "small" | "medium" | "large" | "x-large" | "xx-large" |
-                   "larger" | "smaller" |
-                   CSSLength | CSSPercentage;
```

Comment:
did you mean to remove this indent? can you keep it indented (or move to 4 spaces), please?